apps/api/src/server/app/inspector/inspector.controller.js

```diff
@@ -72,9 +72,9 @@ const formatAppealForAssigningAppeals = function(appeal, reason) {
 		provisionalVisitType: provisionalAppealSiteVisitType(appeal),
 		appealAge: daysBetweenDates(appeal.startedAt, new Date()),
 		appealSite: formatAddressLowerCase(appeal.address),
-		...(reason != undefined && {reason: reason})
-	}
-}
+		...(reason != undefined && { reason: reason })
```

Comment:
1. use strict equals (not sure why eslint isn't enforcing this)
2. it's preferrable to write this as `{ reason }` seeing as prop and variable are the same name